### ISSUE TITLE
io.fabric8.kubernetes.api.model.ManagedFieldsEntry

### ISSUE THREAD

**Issue by sms7182** [kestra-io/kestra#5490]
### Issue description

i want to know about that i don't found this plugin for example , i use image: kestra/kestra:latest but just i could work with two plugins podcreate and apply, i want to use io.fabric8.kubernetes.api.model.ManagedFieldsEntry,

**Comment by Ben8t** [comment]
Hello, `io.fabric8.kubernetes.api.model.ManagedFieldsEntry` is not part of official Kestra plugins (listed [here](https://kestra.io/plugins) - they all start with `io.kestra` prefix). Where did you hear about it ?

**Comment by sms7182** [comment]
> Hello, `io.fabric8.kubernetes.api.model.ManagedFieldsEntry` is not part of official Kestra plugins (listed [here](https://kestra.io/plugins) - they all start with `io.kestra` prefix). Where did you hear about it ?

yes i know, but i don't know about another plugins for example io.fabric8.kubernetes.api.model.
